DOCS-6381-SAML-response-element-and-link (#87)
mulesoft/docs-access-management: modules/ROOT/pages/troubleshoot-saml-assertions-task.adoc

> . In the Chrome developer tool *Form data* window, select the *Params* tab. Other tools or tracers may have different labels.

Reviewer: ```suggestion
. In the Chrome developer tool *Form data* window, select the *Params* tab. 
+
Other tools or tracers might have different labels.
```

. Record the SAML response element.

> After recording the SAML response element, use it to xref:access-management::saml-bearer-token.adoc[obtain a bearer token] to access platform APIs. It is also useful to provide the SAML response for troubleshooting when contacting MuleSoft support.

Reviewer: You record the response, not the element, right?
```suggestion
After recording the SAML response, use it to xref:access-management::saml-bearer-token.adoc[obtain a bearer token] to access platform APIs. It is also useful to record the SAML response to provide to MuleSoft support if you contact them.
```

----

. Provide a screen shot of your SAML configuration in Anypoint Platform.
. Include an XML SAML assertion.

== Example: Chrome Developer Tools SAML Response Element

> The following image shows an instance of Chrome developer tools that contains a SAML response element after logging in to Anypoint Platform using SSO.

Reviewer: This sentence is confusing to me because I am not sure what we are emphasizing: an instance of tools, tools, the response included in the instance or by the tools, or that logging in creates a response...? Please clarify.

Best guess:
```suggestion
The following image shows a SAML response element [in Chrome developer tools/] as a result of logging in to Anypoint Platform using SSO.
```

Reviewer: Also...as a result of YOU logging in?

.SAML response element after logging in to Anypoint Platform in Google Chrome.
image::iam-chromedevtools.png[]

> In the *Network* tab, the `receive-id` line indicates that you logged in to Anypoint Platform using your SSO.

Author: ```suggestion
In the *Network* tab, the `receive-id` line indicates that you attempted to log in to Anypoint Platform using your SSO.
```

> In the *Network* tab, the `receive-id` line indicates that you attempted to log in to Anypoint Platform using your SSO.

Author: ```suggestion
In the *Network* tab (1), the `receive-id` line (2) indicates that you attempted to log in to Anypoint Platform using your SSO.
```

> In the *Network* tab (1), the `receive-id` line (2) indicates that you attempted to log in to Anypoint Platform using your SSO.

Reviewer: "your SSO" or "your SSO ...value?"

Author: ```suggestion
In the *Network* tab (1), the `receive-id` line (2) indicates that you attempted to log in to Anypoint Platform using single sign-on through your identity provider.
```

> When you expand `receive-id`, you can access form data that contains your SAML response element.  

Author: ```suggestion
If the login is successful, you can expand `receive-id` and access form data that contains your SAML response element.  
```

> If the login is successful, you can expand `receive-id` and access form data that contains your SAML response element.  

Author: ```suggestion
The login is successful, and you can expand `receive-id` and access form data that contains your SAML response element (3).  
```

> The login is successful, and you can expand `receive-id` and access form data that contains your SAML response element (3).  

Author: ```suggestion
The login is successful, and you can expand `receive-id` and access form data that contains your SAML response element (3). Note that unsuccessful login attempts still generate SAML response elements.  
```

> The login is successful, and you can expand `receive-id` and access form data that contains your SAML response element (3). Note that unsuccessful login attempts still generate SAML response elements.  

Reviewer: ```suggestion
Because the login is successful, you can expand `receive-id` and access form data that contains your SAML response element (3). Note that unsuccessful login attempts still generate SAML response elements.  
```
So...are these two sentences saying that whether the login is successful or not, you can expand...to see the SAML response? Maybe what we want, if so, is:

```suggestion
In the *Network* tab (1), the `receive-id` line (2) indicates that you attempted to log in to Anypoint Platform using your SSO ID. Whether the login is successful or not, you can expand `receive-id` and access form data that contains your SAML response element (3).
```

Author: Contextually speaking, the first option is more useful.
